CHANGELOG.md

```diff
@@ -5,6 +5,10 @@ This CHANGELOG follows the format listed at [Keep A Changelog](http://keepachang
 
 ## [Unreleased]
 
+## [5.0.1] - 2017-05-09
```

Comment:
Actually the whole line, if it was not gonna change the date in 2.5 minutes I would have accepted it.